src/app/@ansyn/overlays/components/overlay-overview/overlay-overview.component.less

```diff
@@ -0,0 +1,46 @@
+@import '../../../core/less/common';
```

Comment:
`core/less/common`.... ⬆️

Reply:
So why... it just gives me errors in Webstorm...

Reply:
I know, web storm is not synchronized with angular-cli.json
but relative path breaking builder